govcd/vapp.go

```diff
@@ -389,6 +390,23 @@ func (vapp *VApp) Suspend() (Task, error) {
 		"", "error suspending vApp: %s", nil)
 }
 
+// DiscardSuspendedState takes back a vApp from suspension
+func (vapp *VApp) DiscardSuspendedState() error {
+	if vapp.VApp.Status != 3 {
```

Comment:
I think we don't have defined constants for these "magic" numbers, so maybe we could add a comment similar to govcd/vapp.go:224?

Reply:
We have a map that should be reversed to be of any use here (`types.VAppStatuses`), so I added a comment.